frontend/src/app/modules/authentication/registration/registration.component.html

```diff
@@ -10,35 +10,53 @@ <h4>Sign Up</h4>
 
         <app-barrier [text]="'or'"></app-barrier>
 
-        <form [formGroup]="registerForm" (ngSubmit)="registerForm.valid && register()" class="registration-form"
+        <form
+            [formGroup]="registerForm"
+            (ngSubmit)="registerForm.valid && register()"
+            class="registration-form"
             action="">
-
-            <app-input [formControl]="$any(registerForm.controls['username'])" [label]="'username'"
+            <app-input
+                [formControl]="$any(registerForm.controls['username'])"
+                [label]="'username'"
                 placeholder="Enter your Username">
             </app-input>
-            <app-input [formControl]="$any(registerForm.controls['email'])" [label]="'email'"
+            <app-input
+                [formControl]="$any(registerForm.controls['email'])"
+                [label]="'email'"
                 placeholder="Enter your Email">
             </app-input>
-            <app-input [formControl]="$any(registerForm.controls['firstName'])" [label]="'first Name'"
+            <app-input
+                [formControl]="$any(registerForm.controls['firstName'])"
+                [label]="'first Name'"
                 placeholder="Enter your FirstName">
             </app-input>
-            <app-input [formControl]="$any(registerForm.controls['lastName'])" [label]="'last Name'"
+            <app-input
+                [formControl]="$any(registerForm.controls['lastName'])"
+                [label]="'last Name'"
                 placeholder="Enter your LastName">
             </app-input>
-            <app-input [formControl]="$any(registerForm.controls['password'])" [label]="'password'" [type]="'password'"
+            <app-input
+                [formControl]="$any(registerForm.controls['password'])"
+                [label]="'password'"
+                [type]="'password'"
                 placeholder="Enter your Password">
             </app-input>
-            <app-input [formControl]="$any(registerForm.controls['confirmPassword'])" [label]="'confirm Password'"
-                [type]="'password'" placeholder="Enter your Password again">
+            <app-input
+                [formControl]="$any(registerForm.controls['confirmPassword'])"
+                [label]="'confirm Password'"
```

Comment:
You can write
label="confirm Password", no need for brackets

Reply:
fixed